### DISCUSSION TITLE
very nice effective stable vpnkit

### DISCUSSION THREAD

**Discussion by tianfudhe** [sakai135/wsl-vpnkit#152] [General]
recently my work relies on a bunch of vpns. but I get used to WSL. 
before using this vpnkit, I'm struggling when choosing windows-native dev environment and WSL.

but now, I feel free to use WSL quite a lot. other networks like linux docker also run the networks well.

Thank the wsl-vpnkit developers so much!

**Comment by MicheleBortol** [comment]
Hi,

Just commenting here to say this tool worked out of the box for me. Really an amazing job!
A big thank you to the devs!

Michele

**Comment by mavaddat** [comment]
Same here! I am behind ZScaler and Pulse Secure VPN, so it is incredibly painful for me to get connected to anything on my corporate laptop. This is a life-saver.

**Comment by MitchIonascu** [comment]
Thank you, amazing work! This allowed me to use WSL2 as my daily driver, the speed compared to WSL1 is great!
